index.bs

```diff
@@ -269,6 +272,9 @@ To <dfn id="waiting-for-all" export lt="wait for all|waiting for all">wait for a
 1. Let |index| be 0.
 1. Let |total| be |promises|'s [=list/size=].
 1. Let |result| be a <a>list</a> containing |total| null values.
+1. If |total| is 0, then:
+    1. <a>Queue a microtask</a> to perform |successSteps| given |result|.
```

Comment:
I hope I'm doing this [autolinking](https://tabatkins.github.io/bikeshed/#autolinking) right! 😅

I still don't know when to use "dfn autolinks" (`[= ... =]`) and when to use manual `<a>` elements though... 🤷‍♂

Reply:
They're equivalent. Basically `[=...=]` is newer so some older specs like this one still use `<a>`. (And also sometimes newer specs written by people with older habits.)